packages/sdk-v2/src/sdk/goodid/db/createDb.ts

```diff
@@ -0,0 +1,20 @@
+import Dexie from "dexie";
+
+/**
+ * Schema definition for the certificates collection
+ * - `id`: incrementing id linked to various verified credentials
+ * Schema definition for the certificates collection (A collection of credentials per issuer)
+ * - `type`: should indicate the types of the credential
+ * - Other fields are according to Veramo's/W3C Credential schema
+ */
+
+const schema = {
+  certificates: "++id",
+  certificates_credentials: "++index, certificate_id, credentialSubject_id, issuer_id, type, issuanceDate"
```

Comment:
what are these two types?
i've ask for super super simple solution

Reply:
changes requested by @johnsmith-gooddollar but not the final fixed version of what he wants